index.html

```diff
@@ -42,5 +42,119 @@
 
     </header>
 
-  </body>
+    <main class="main-index">
+      <h1 class="visually-hidden">Интернет-магазин device</h1>
+      <img class="logo" src="#" alt="квадрокоптер на желто-белом фоне">
+    </main>
+    <section class="advantages">
+      <h2 class="visually-hidden">Преимущества</h2>
+      <span>Порхает как бабочка, жалит как пчела!</span>
+      <p>Этот обычный, на первый взгляд, квадрокоптер оснащен мощным лазером, замаскированным под стандартную камеру.</p>
+      <a href="#"><p>Подробнее</p></a>
+      <span>800 м</span>
+      <span>Дальность полета</span>
+      <span>50 м</span>
+      <span>Радиус поражения</span>
+    </section>
+    <ul>
+      <li>
+        <img class="logo" src="#" alt="очки виртуальной реальности">
+        <p>Виртуальная реальность</p>
+      </li>
+        <li>
+        <img class="logo" src="#" alt="монопод для селфи">
+        <p>Моноподы для селфи</p>
+      </li>
+        <li>
+        <img class="logo" src="#" alt="экшн-камера">
+        <p>Экшн-камеры</p>
+      </li>
+        <li>
+        <img class="logo" src="#" alt="фитнес-браслет">
+        <p>Фитнес-браслеты</p>
+      </li>
+        <li>
+          <img class="logo" src="#" alt="умные часы">
+        <p>Умные часы</p>
+      </li>
+        <li>
+        <img class="logo" src="#" alt="квадрокоптер">
+        <p>Квадрокоптеры</p>
+      </li>
+    </ul>
+<section class="delivery">
+  <ul>
+    <li>Доставка</li>
+    <li>Гарантия</li>
+    <li>Кредит</li>
+  </ul>
+    <h2>Доставка</h2>
+    <p>Мы с удовольствием доставим ваш товар прямо
+к вашему подъезду совершенно бесплатно! Ведь
+мы неплохо заработаем, поднимая его на ваш этаж.</p>
+<img class="" src="#" alt="машина">
+<p>Доставим редкий товар под заказ</p>
+</section>
+<section class="appointment">
+<h2>О нас</h2>
+<p>Огромный выбор гаджетов не оставит равнодушным гика, который есть в каждом из нас.
+</p>
+<p>Мы можем доставить ваш товар в самые отдалённые уголки России транспортными компаниями:
+</p>
+<ul>
+  <li>Меловые линии</li>
+  <li>СДИК</li>
+  <li>ТоксБерри</li>
+</ul>
+<button class="button">Подробнее о нас</button>
```

Comment:
```suggestion
<a href="#" class="button">Подробнее о нас</a>
```
Ведь это переход куда-то.
Действие = кнопка
Переход = ссылка